Dockerfile

```diff
@@ -7,6 +7,6 @@ WORKDIR /app
 COPY . .
 RUN make build
 
-FROM registry.fedoraproject.org/fedora-minimal:38
+FROM cgr.dev/chainguard/glibc-dynamic:latest
```

Comment:
Can we use something else than `latest`? Mainly in order to get reproducible builds, but also to have more control on what we publish.

Unfortunately, I think that `cgr.dev/chainguard/glibc-dynamic` does not have SemVer tags, then we would need to use the digest. We would need to manually update the images e.g. each week (side note: AFAIK this approach is actually seen as a better practice).

References: 
- https://vsupalov.com/docker-latest-tag/
- https://medium.com/@michael.vittrup.larsen/why-we-should-use-latest-tag-on-container-images-fc0266877ab5
- https://hackernoon.com/docker-images-name-vs-tag-vs-digest